.github/workflows/test-cargo.yml

```diff
@@ -100,6 +100,7 @@ jobs:
         run: |
           pnpx node-gyp install
           choco install protoc -y
+          echo 'node-linker = "hoisted"' > crates/turbopack/tests/node-file-trace/.npmrc
```

Comment:
Can we leave short comment for why / what?

Reply:
Yes, I'm writing the linked issue...

Reply:
https://github.com/vercel/turbo/issues/2622

Reply:
I think it's worth to leave a comment in the yml itself, but not a blocker.